front/components/poke/subscriptions/table.tsx

```diff
@@ -46,9 +46,15 @@ function prepareSubscriptionsForDisplay(
       planCode: s.plan.code,
       status: s.status,
       startDate: s.startDate
-        ? new Date(s.startDate).toLocaleDateString()
+        ? `${new Date(s.startDate).toLocaleDateString()} ${new Date(
```

Comment:
We could also just use `toLocaleString()` that would print `11/15/2023, 5:33:49 PM`